src/mudlet-lua/lua/Other.lua

```diff
@@ -145,6 +145,8 @@ local group_creation_functions = {
   alias = function(name, parent)
     return not (permAlias(name, parent, "", "") == -1)
   end,
+  key = function(name, parent)
+    return not (permKey(name, parent, -1, "") == -1)
```

Comment:
```suggestion
  key = function(name, parent)
    return not (permKey(name, parent, -1, "") == -1)
  end,
```
without this I doesn't even load the file for me.

Reply:
This must have come after my original commit, no idea how. Of course, will be fixed.